modules/logging_config/log.sh

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+# TODO: idk what the channel names are
+CHANNELS=(can_primary can_secondary)
+declare -A PIDS
+CURR_MP=""
+
+timestamp() {
+  awk '{ print strftime("[%Y-%m-%d %H:%M:%S]"), $0; fflush(); }'
+}
+
+wait_for_mount() {
+  local devnode="$1" mp=""
+  while :; do
+    mp="$(lsblk -no MOUNTPOINT "$devnode" | grep -m1 . || true)"
+    [[ -n "$mp" ]] && { echo "$mp"; return; }
+    sleep 1
+  done
+}
+
+init_dump() {
+  CURR_MP="$1"
+  for ch in "${CHANNELS[@]}"; do
+    local dir="$CURR_MP/$ch"
+    mkdir -p "$dir"
+    local file="$dir/$(date +"%Y-%m-%d_%H-%M-%S.log")"
+    echo "[can-log] starting candump on $ch into $dir/"
+    # (candump -l "$ch" | timestamp>"$dir/")
+
+    ( yes "$ch dummy log line" | timestamp>"$file" ) &
```

Comment:
I wouldn't manually add timestamps like this. It might make it so that cantools can't parse the logged data?